auraed/src/runtime/mod.rs

```diff
@@ -236,7 +238,8 @@ impl CellService {
             .cpu()
             .shares(cpu_shares)
             .mems(cpu_mems)
-            .quota(cpu_quota)
+            .period(1000000) // microseconds in a second
+            .quota(*cpu_quota)
```

Comment:
I think this copies the i64, which is cheap, but we could use `cpu_quota.into_inner()` to avoid the copy.